pkg/utils/constants.go

```diff
@@ -43,3 +43,6 @@ const ServiceInstanceKind = "ServiceInstance"
 
 // ServiceBindingKind a constant representing the kind of the Service Binding model
 const ServiceBindingKind = "ServiceBinding"
+
+// ServiceAccountKind a constant representing the kind of the ServiceAccount Model
+const ServiceAccountKind = "ServiceAccount"
```

Comment:
Has this been added to the API model? The API model will need a kind attribute and we need to make sure that it is set to this value.